src/lib/tour-overlay/TourOverlay.component.tsx

```diff
@@ -131,44 +142,42 @@ export const TourOverlay = React.forwardRef<TourOverlayRef, TourOverlayProps>((p
       }),
       Animated.timing(tipOpacity, {
         delay: 500,
-        duration: 500,
+        duration: 300,
```

Comment:
Should the user have the capabilities to extend those animation timings at some point? I think not but I'm mentioning it for further considerations.

Reply:
Maybe we can find a way to do it later on. It's a good improvement for the future, but it needs careful planning. The duration/delay times of the animation need to be carefully synced, otherwise, the tooltip will render before the spotlight reaches its final position, causing the tooltip to render in the wrong place.

If you're interested in tackling that improvement, feel free to open an issue or PR. It'll be a great addition to the library 🙂